simplq/src/components/pages/Join/index.jsx

```diff
@@ -46,7 +46,9 @@ export default (props) => {
     <div>
       <HeaderSection queueName={queueStatusResponse.queueName} history={props.history} />
       <div className={styles['main-content']}>
-        <QueueStats queueStatus={queueStatusResponse} />
+        <div style={{ display: 'flex', justifyContent: 'center' }}>
+          <QueueStats queueStatus={queueStatusResponse} />
```

Comment:
Could you create a class for this style and use it from the join SCSS file? We're separating styles from components everywhere else.

Reply:
That was an unfortunate commit 😊